docs/governance/active.md

```diff
@@ -0,0 +1,16 @@
+## Active Governance Proposals
+
+The Proposals listed below have been approved and are currently in effect:
+
+### No deadline
+
+- [HGP-3 [GC] Change Minimum Quorum for approval to % of Circulating Supply](https://snapshot.org/#/hbot.eth/proposal/0x9a58b2136b5c20af9d4fd507cd8174e7f6e21425549bbe864e089154f76da415)
+
+### End of Epoch 1 (June 30, 2022)
+
+- [HGP-3 [GC] Change Minimum Quorum for approval to % of Circulating Supply](https://snapshot.org/#/hbot.eth/proposal/0x9a58b2136b5c20af9d4fd507cd8174e7f6e21425549bbe864e089154f76da415)
```

Comment:
Remove this one since it's already listed above

Reply:
Was a typo. Corrected to HGP-2 (Miner distribution)